library/common/data/utility.cc

```diff
@@ -20,8 +28,9 @@ Buffer::InstancePtr toInternalData(envoy_data data) {
   return buf;
 }
 
-envoy_data toBridgeData(Buffer::Instance& data) {
-  envoy_data bridge_data = copyToBridgeData(data);
+envoy_data toBridgeData(Buffer::Instance& data, uint32_t max_bytes) {
```

Comment:
This is an odd name for a type. Does Envoy Mobile use a different style guide? (not that this has anything to do with your PR)

Reply:
AFIK the bridge is c structures so yeah different style for that part

Reply:
That's correct - we currently have standardized on the C bridge types (both structs and aliases) using snake case. It's not set in stone, but it helps to distinguish them.